scripts/wrapped_geth_dev.sh

```diff
@@ -21,6 +21,13 @@ main() {
         exit 1
     fi
 
+    # CONCURRENT_BLOCK_FLUSHING=true ./scripts/run_firehose_geth_dev.sh 3.0 prague
+    concurrent_block_flushing="${CONCURRENT_BLOCK_FLUSHING:-0}"
```

Comment:
Now with a count, the variable is really not aligned anymore, we will need to find a better name as there is actually  no concurrent flushing.

The value controls are many worker pick from the queue correct?